chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/DeveloperManager.java

```diff
@@ -178,7 +178,7 @@ public DeveloperDTO getById(Long id) throws EntityRetrievalException {
     @Transactional(readOnly = false)
     @CacheEvict(value = {
             CacheNames.ALL_DEVELOPERS, CacheNames.ALL_DEVELOPERS_INCLUDING_DELETED, CacheNames.COLLECTIONS_DEVELOPERS,
-            CacheNames.GET_DECERTIFIED_DEVELOPERS
+            CacheNames.GET_DECERTIFIED_DEVELOPERS, CacheNames.DEVELOPER_NAMES
```

Comment:
I think you need a `CacheNames.COLLECTIONS_LISTINGS` here too

Reply:
Added